dll/win32/shell32/lang/pt-BR.rc

```diff
@@ -48,8 +48,8 @@ BEGIN
     MENUITEM SEPARATOR
     MENUITEM "Cu&t\tCtrl+X", FCIDM_SHVIEW_CUT
     MENUITEM "&Copy\tCtrl+C", FCIDM_SHVIEW_COPY
-    MENUITEM "&Paste\tCtrl+V", FCIDM_SHVIEW_INSERT
-    MENUITEM "Paste &shortcut", FCIDM_SHVIEW_INSERTLINK
+    MENUITEM "Co&lar\tCtrl+V", FCIDM_SHVIEW_INSERT
+    MENUITEM "Colar a&talho", FCIDM_SHVIEW_INSERTLINK
```

Comment:
This would introduce an accelerator collision between "a&talho" and "Cu&t", You should seek for advice from brazilan guy how to solve that. But you shouldn't introduce a collision.